lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1024,7 +1024,7 @@ def restart_all_services(self):
         """Restart all magma services on magma_dev VM"""
         if self._init_system == InitMode.SYSTEMD:
             self.exec_command(
-                "sudo systemctl stop 'magma@*' 'sctpd' ;"
+                "sudo systemctl stop 'magma@*' ;"
```

Comment:
Did you figure out, why the `sctpd` should not be restarted here anymore?

Reply:
As far as I remember stopping clears redis in some very convoluted way. 
1. sctpd gets stopped  here
2. sctpd get triggerd by sudo systemctl start magma@magmad (via systemd dependency)
3. sctpd is started
4. in the sctpd systemd service ExecStartPre=/usr/bin/env python3 /usr/local/bin/config_stateless_agw.py sctpd_pre triggers clear_redis_state

Without cleaning the redis here I guess a lot of new interdependencies would show up int tests.

Reply:
@mpfirrmann what do you think you have looked deeper into this.

Reply:
@crasu Note that the function `restart_all_services` is only called in a few places: when re-enabling nat and in three "tests" that modify config: `test_modify_config_for_non_sanity`, `test_modify_mme_config_for_sanity` and `test_restore_config_after_non_sanity`. Do we need/want sctpd to be restarted in these situations?

Reply:
This test will definitly fail without the restart `test_modify_config_for_non_sanity`.